src/main/java/frc/robot/Robot.java

```diff
@@ -1,109 +1,137 @@
+  private Command m_autonomousCommand;
+  private RobotContainer m_robotContainer;
+
+  /**
+   * This function is run when the robot is first started up and should be used for any
+   * initialization code.
+   */
+
+  @Override
+  public void robotInit() {
+    // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
+    // autonomous chooser on the dashboard.
+    m_robotContainer = new RobotContainer();
+    AddressableLED m_led = new AddressableLED(9); // PWM port 9
+    AddressableLEDBuffer m_ledBuffer = new AddressableLEDBuffer(150); // Our full LED strip length
+    m_led.setLength(m_ledBuffer.getLength());
+
+    for (int i = 0; i < m_ledBuffer.getLength(); i++) {
+      // Sets the specified LED to the RGB values for red
+      m_ledBuffer.setRGB(i, 119,15,5);
+    }
+    m_led.setData(m_ledBuffer);
+    m_led.start();
+    //CameraServer.startAutomaticCapture();
+  }
+
+  /**
+   * This function is called every 20 ms, no matter the mode. Use this for items like diagnostics
+   * that you want ran during disabled, autonomous, teleoperated and test.
+   *
+   * <p>This runs after the mode specific periodic functions, but before LiveWindow and
+   * SmartDashboard integrated updating.
+   */
+  @Override
+  public void robotPeriodic() {
+    // Runs the Scheduler.  This is responsible for polling buttons, adding newly-scheduled
+    // commands, running already-scheduled commands, removing finished or interrupted commands,
+    // and running subsystem periodic() methods.  This must be called from the robot's periodic
+    // block in order for anything in the Command-based framework to work.
+    CommandScheduler.getInstance().run();
+  }
+
+  /** This function is called once each time the robot enters Disabled mode. */
+  @Override
+  public void disabledInit() {}
+
+  @Override
+  public void disabledPeriodic() {}
+
+  /** This autonomous runs the autonomous command selected by your {@link RobotContainer} class. */
+  @Override
+  public void autonomousInit() {
+    m_autonomousCommand = m_robotContainer.getAutonomousCommand();
+    // schedule the autonomous command (example)
+    if (m_autonomousCommand != null) {
+      m_autonomousCommand.schedule();
+    }
+  }
+
+  /** This function is called periodically during autonomous. */
+  @Override
+  public void autonomousPeriodic() {
+      SmartDashboard.putNumber("Left Encoder", m_robotContainer._drive_Train.getLeftEncoder());
+      SmartDashboard.putNumber("Right Encoder", m_robotContainer._drive_Train.getRightEncoder());
+  }
+
+  @Override
+  public void teleopInit() {
+    // This makes sure that the autonomous stops running when
+    // teleop starts running. If you want the autonomous to
+    // continue until interrupted by another command, remove
+    // this line or comment it out.
+    if (m_autonomousCommand != null) {
+      m_autonomousCommand.cancel();
+    }
+
+  
+    m_robotContainer._drive_Train.setDefaultCommand(

```

Comment:
hey I actually don't like that we did this, we should put it back to the way it was before that guy on the other team "helped" us at the San Antonio competition

Reply:
fixed